pages/covid.js

```diff
@@ -46,14 +50,14 @@ function COVID() {
           <a className="button" href="https://docs.google.com/document/d/1IcBZ4yj4kJ14Gg0PVw3xbg-QqRYUQGUlHm8vVskgQOQ/edit?usp=sharing" target="_blank" rel="noopener noreferrer">
             <FontAwesomeIcon icon={faExternalLinkAlt} fixedWidth aria-hidden={true} /> Link to Google Doc
           </a>{' '}
-          <a className="button" href="/files/acm-covid-policies-s22-last-updated-03-27-22.pdf" target="_blank" rel="noopener noreferrer">
+          <a className="button" href="/files/acm-covid-policies-s22-last-updated-04-11-22.pdf" target="_blank" rel="noopener noreferrer">
             <FontAwesomeIcon icon={faFilePdf} fixedWidth aria-hidden={true} /> Link to PDF
           </a>
         </div>
         <br />
         <iframe
           title="ACM COVID Policies Spring 2022"
-          src="https://docs.google.com/document/d/e/2PACX-1vQIXLd4Omqv2XaDflpIlbmyfYyXC0GhE64XbS1kakNOHf-9PauJnIBTiNFz_rEQteixMecfN8fKld5v/pub?embedded=true"
+          src="https://docs.google.com/document/d/1IcBZ4yj4kJ14Gg0PVw3xbg-QqRYUQGUlHm8vVskgQOQ/edit?usp=sharing"
```

Comment:
For the embedded iframe, you have to use a special link made by publishing a google docs page to the web, here's the correct link! https://docs.google.com/document/d/e/2PACX-1vQIXLd4Omqv2XaDflpIlbmyfYyXC0GhE64XbS1kakNOHf-9PauJnIBTiNFz_rEQteixMecfN8fKld5v/pub?embedded=true